modules/programs/aerospace.nix

```
@@ -269,9 +296,28 @@ in
 
     home = {
       packages = lib.mkIf (cfg.package != null) [ cfg.package ];
+
       file.".config/aerospace/aerospace.toml".source = tomlFormat.generate "aerospace" (
-        filterNulls cfg.userSettings
+        filterNulls (
+          cfg.userSettings
+          // {
+            # Override these to avoid launchd conflicts
+            start-at-login = false;
+            after-login-command = [ ];
+          }
+        )
```

```suggestion
          // lib.optionalAttrs cfg.launchd.enable {
            # Override these to avoid launchd conflicts
            start-at-login = false;
            after-login-command = [ ];
          }
        )
```

I think we should add a note about this behavior to the `launchd` option as well. Just to avoid surprising users.

---

how can I add this note?

---

Just to the `launchd.enable` description somewhere mentioning this particular side effect. 

Something similar to "NOTE: This will disable `aerospace` managing its own launchd agent, by configuring `start-at-login` and `after-login-command`."